projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -227,24 +227,9 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
      * Controls the copy behavior of the grid.
      */
     @Input()
-    public clipboardOptions = {
-        /**
-         * Enables/disables the copy behavior
-         */
-        enabled: true,
-        /**
-         * Include the columns headers in the clipboard output.
-         */
-        copyHeaders: true,
-        /**
-         * Apply the columns formatters (if any) on the data in the clipboard output.
-         */
-        copyFormatters: true,
-        /**
-         * The separator used for formatting the copy output. Defaults to `\t`.
-         */
-        separator: '\t'
-    };
+    public get clipboardOptions(): ClipboardOptions {
```

Comment:
Why introduce a getter? There's no need for a custom setter for this property. Just introduce it as a property with default value.